crates/gh-workflow-tailcall/src/workflow.rs

```diff
@@ -115,8 +118,8 @@ fn release_job(cond: &Context<bool>, build: &Job, permissions: &Permissions) ->
         .add_step(Release::default().command(Command::Release))
 }
 
-fn build_and_test() -> Job {
-    Job::new("Build and Test")
+fn build_and_test(value: &Workflow) -> Job {
```

Comment:
```suggestion
fn build_and_test(&self) -> Job {
```
Convert to a method in Workflow

Reply:
@coderabbitai resolve

Reply:
Can you create a suggestion for the fix @coderabbitai

Reply:
@coderabbitai: Can you push commit with this changes?